### ISSUE TITLE
[FEAT] pre-deploy scripts

### ISSUE THREAD

**Issue by hevans90** [flowaccount/nx-plugins#30]
**Is your feature request related to a problem? Please describe.**
Node/Typescript serverless lambda: So I have been playing around with your plugin (it's great!) and trying to use libraries in conjunction with a simple node lambda function that I am deploying to AWS.

My use case is that I want to use a library like `bcrypt` that webpack registers as an `external`, meaning the dependency is gathered as a part of your `deploy` script, not `build`.

So running `nx build my-lib` will only generate a single JavaScript file in dist, but running `nx deploy my-lib` will pull in the externals inside `dist/my-lib/node_modules`.

This all sounds great until one of these externals requires building with something like `node-pre-gyp` before deployment to AWS. Right now there is no way of say... going in to `dist/my-lib/node_modules/bcrypt` and running the build command to generate the .node binary necessary for the lambda to actually run.

**Describe the solution you'd like**
An optional property adding to the builder to specify pre-deploy scripts that happen **after** webpack is done compiling the bundle.

**Check which provider is affected:**
[*] AWS
[] Azure
[] Google Cloud Platform

**Check which framework is affected:**
[] Angular
[*] Nodejs
[] Serverless
[] Lambda
[] Infrastructure as a code

**Comment by wickstargazer** [comment]
We are actually facing the same issue of having to build some static libraries before deploying! ... I would be looking into it over a few coming weeks. 

For now, if you want to just package without deploy, try using the `sls` command? 

Also if you have time to help I would appreciate it a lot :smile:

**Comment by wickstargazer** [comment]
added option to allow this now.
```
    "ignoreScripts": {
      "type": "boolean",
      "default": true,
      "description": "Updates ignoreScripts false to use own package with dev environment"
    }
```

**Comment by wickstargazer** [comment]
fixed in v0.5.3
